.circleci/config.yml

```diff
@@ -34,56 +32,147 @@ jobs:
+                 DEPLOYMENT=$(cat deploy/deployreq.json | jq '.payload.kubernetes.resources += ['$NAISERATOR']')
+                 DEPLOYMENT=$(echo $DEPLOYMENT | jq '.environment = "prod-fss"')
+                 DEPLOYMENT=$(echo $DEPLOYMENT | jq '.ref = "'$COMMIT_SHORT'"')
+                 curl -u "$GITHUB_USERNAME:$GITHUB_PASSWORD" --fail \
+                    -X POST \
+                    -d "$DEPLOYMENT" \
+                    -H "Accept: application/vnd.github.ant-man-preview+json" \
+                    https://api.github.com/repos/navikt/helse-speil/deployments
+
+   deploy-redis-to-preprod:
+      working_directory: ~/speil
+      docker:
+         - image: circleci/node:12.2.0-stretch
+      steps:
+         - setup_remote_docker:
+              docker_layer_caching: true
+         - run:
+              name: deploy redis to preprod
+              command: |
+                 set -e
+                 docker create -v /nais --name redispreprod alpine:3.4 /bin/true
+                 docker cp deploy/redis.yaml redispreprod:/nais
+                 NAISERATOR=$(docker run --volumes-from redispreprod mikefarah/yq yq r /nais/redis.yaml -j)
+                 DEPLOYMENT=$(cat deploy/deployreq.json | jq '.payload.kubernetes.resources += ['$NAISERATOR']')
+                 DEPLOYMENT=$(echo $DEPLOYMENT | jq '.ref = "speil-redis"')
+                 DEPLOYMENT=$(echo $DEPLOYMENT | jq '.environment = "dev-fss"')
+                 curl -u "$GITHUB_USERNAME:$GITHUB_PASSWORD" --fail \
+                       -X POST \
+                       -d "$DEPLOYMENT" \
+                       -H "Accept: application/vnd.github.ant-man-preview+json" \
+                       https://api.github.com/repos/navikt/helse-speil/deployments
+
+   deploy-redis-to-prod:
+      working_directory: ~/speil
+      docker:
+         - image: circleci/node:12.2.0-stretch
+      steps:
+         - setup_remote_docker:
+              docker_layer_caching: true
+         - run:
+              name: deploy redis to prod
+              command: |
+                 set -e
+                 docker create -v /nais --name redisprod alpine:3.4 /bin/true
+                 docker cp deploy/redis.yaml redispreprod:/nais
+                 NAISERATOR=$(docker run --volumes-from redisprod mikefarah/yq yq r /nais/redis.yaml -j)
+                 DEPLOYMENT=$(cat deploy/deployreq.json | jq '.payload.kubernetes.resources += ['$NAISERATOR']')
+                 DEPLOYMENT=$(echo $DEPLOYMENT | jq '.ref = "speil-redis"')
+                 DEPLOYMENT=$(echo $DEPLOYMENT | jq '.environment = "prod-fss"')
+                 curl -u "$GITHUB_USERNAME:$GITHUB_PASSWORD" --fail \
+                       -X POST \
+                       -d "$DEPLOYMENT" \
+                       -H "Accept: application/vnd.github.ant-man-preview+json" \
+                       https://api.github.com/repos/navikt/helse-speil/deployments
+
+workflows:
```

Comment:
Kan man ikke filtrere på workflow-nivå også?
```
workflows:
   version: 2
   master:
      filters:
         branches:
            only: master
      jobs:
         - deploy-redis-to-preprod
         - deploy-redis-to-prod
         - build-speil
         - deploy-speil-to-preprod:
            requires:
               - build-speil
               - deploy-redis-to-preprod
         - deploy-speil-to-prod:
            requires:
               - build-speil
               - deploy-redis-to-prod
   not_master:
      filters:
         branches:
            ignore: master
      jobs:
         - build-speil
```

Reply:
Det skulle man tro, men Circle sin cli sier nei: `Error: Config does not conform to schema`

Reply:
Ok, jeg lot meg inspirere herfra: https://circleci.com/docs/2.0/configuration-reference/#filters-1

Det var ikke noe med indentering, da?

Reply:
https://ideas.circleci.com/ideas/CCI-I-258

`On roadmap`

Reply:
Ah, kjipern. Ble litt fintet ut av dokumentasjonen deres, de nevner jo filtrering på workflow-nivå flere steder, men det gjelder kanskje i forbindelse med `schedule`-konfig, da.